ClosedXML/Excel/XLWorkbook.cs

```diff
@@ -640,7 +640,7 @@ public IXLCells FindCells(Func<IXLCell, Boolean> predicate)
 
         public IXLRows FindRows(Func<IXLRow, Boolean> predicate)
         {
-            var rows = new XLRows(null);
+            var rows = new XLRows(null, null);
```

Comment:
Please use named parameters in these cases. I realise it was like that already, but since we're touching lines, we might as well fix it.